src/zzz.ts

```diff
@@ -0,0 +1,26 @@
+const completionSpec: Fig.Spec = {
+  name: "zzz",
+  description: "",
```

Comment:
Is this the correct description?
```suggestion
  description: "A command-line MacOS sleep program (like SleepNow)",
```